metricbeat/docs/modules/http.asciidoc

```diff
@@ -42,5 +42,9 @@ The following metricsets are available:
 
 * <<metricbeat-metricset-http-json,json>>
 
+* <<metricbeat-metricset-http-server,server>>
```

Comment:
this was a wrong checkin from another metricset :) will follow in a subsequent PR. i have removed it from this one.